var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -228,54 +228,55 @@ def install(self, spec, prefix):
 
         # take step-40 which can use both PETSc and Trilinos
         # FIXME: switch step-40 to MPI run
-        with working_dir('examples/step-40'):
-            print('=====================================')
-            print('========== Step-40 PETSc ============')
-            print('=====================================')
-            # list the number of cycles to speed up
-            filter_file(r'(const unsigned int n_cycles = 8;)',
```

Comment:
Step-40 does exist in 8.3.0: https://www.dealii.org/8.3.0/doxygen/deal.II/step_40.html
this part (Step-40 PETSc) should work just fine.